extensions/management.py

```diff
@@ -13,6 +13,37 @@ async def topic(self, ctx, description):
         await ctx.channel.edit(topic=description)
         await ctx.send("Channel topic updated")
 
+    # Listener pour la création des salons vocaux de réunion (et leur supression)
+    @commands.Cog.listener()
+    async def on_voice_state_update(self, member, before, after):
+        # Création
+        try:
+            if (
+                after.channel.name
+                == self.bot.config["voicechannelcreation"]["triggerchannel_name"]
+                and not member.bot
+            ):
+                newchannel = await member.guild.create_voice_channel(
+                    self.bot.config["voicechannelcreation"]["newchannel_name"],
+                    overwrites=None,
+                    category=after.channel.category,
+                )
+                await member.move_to(newchannel)
+        except:

```

Comment:
don't use try except without handling it, it will shadows error in logs, just remove both try except